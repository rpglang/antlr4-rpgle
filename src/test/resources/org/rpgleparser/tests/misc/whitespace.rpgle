     C
     D
     F
     I
     O