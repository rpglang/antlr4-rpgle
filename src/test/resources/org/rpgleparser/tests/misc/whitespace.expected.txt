/*===TOKENS===*/
OS_FIXED                 <O>
'                                                                           '<                                                                           >
HIDDEN:OS_Comments       <                  >
/*===TREE===*/
(r
  (statement
    (blank_spec
      O
      '                                                                           '
      <EOF>
    )
  )
)
/*======*/